src/index.js

```diff
@@ -15,9 +15,18 @@ if (process.env.NODE_ENV !== "production") {
 
 const mapConfig = parseUriHash(defaultMapConfig);
 
-new LeafletMap(mapConfig);
+const mapContainer = new LeafletMap(mapConfig);
 new InfoWindow();
 new TitleDetails();
 new LoadingIndicator();
 
-getData();
+const {
```

Comment:
This seems like a good way to make things dynamic.

Reply:
Would be good to make this a public method on the class instance IMO.